src/runtime/workspace_pool.cc

```diff
@@ -115,7 +115,7 @@ class WorkspacePool::Pool {
   }
   // Release all resources
   void Release(Device dev, DeviceAPI* device) {
-    ICHECK_EQ(allocated_.size(), 1);
+    // ICHECK_EQ(allocated_.size(), 1);
```

Comment:
Could you elaborate why this check was commented out? We may need to remove it if needed, rather than leave the comment in.

Reply:
I think there may be more than 1 if there are multiple device workspaces, e.g., both FPGA and CPU devices. In this case, the check will fail.